azure-pipelines.yml

```diff
@@ -462,7 +487,8 @@ jobs:
       csproj: src/benchmarks/micro/MicroBenchmarks.csproj
       runCategories: 'runtime libraries'
       channels: # for private jobs we want to benchmark .NET Core 5.0 and 3.1
-        - master
+        #- master
+        - release/5.0.1xx
```

Comment:
big thanks for adding `release/5.0.1xx`. I am not sure about commenting out `master` (6.0 early bits). It would be nice if we could have at least 1 windows and 1 Linux CI leg that runs it (just to make sure we can still run the benchmarks for it)

Reply:
Yes after talking with Drew offline we've decided to keep the master branch by just updating the sdk version.

Reply:
> by just updating the sdk version

do you mean the `Microsoft.Dotnet.Sdk.Internal` version? does it mean that Arcade is going to download 6.0 SDK but then the python script is going to download `release/5.0.1xx` and verify the benchmarks against `release/5.0.1xx` SDK?

Reply:
If my thought is right, for master jobs, our ```ci_setup.py``` script is going to download the ```Microsoft.Dotnet.Sdk.Internal``` version 6.0 SDK (after this PR got checked in to unblock maestro update). The script will download from channel ```release/5.0.1xx``` for ```release/5.0.1xx``` jobs. Will uncomment the master jobs in yml.